packages/mjml-button/src/index.js

```diff
@@ -1,141 +1,190 @@
+const endingTag             = true;
 const defaultMJMLDefinition = {
-  content: '',
+  content:    '',
   attributes: {
-    "background-color": "#414141",
-    "border": "none",
-    "border-bottom": null,
-    "border-left": null,
-    "border-radius": "3px",
-    "border-right": null,
-    "border-top": null,
-    "container-background-color": null,
-    "font-style": null,
-    "font-size": "13px",
-    "font-weight": "normal",
-    "font-family": "Ubuntu, Helvetica, Arial, sans-serif",
-    "color": "#ffffff",
-    "text-decoration": "none",
-    "text-transform": "none",
-    "align": "center",
-    "vertical-align": "middle",
-    "href": null,
-    "inner-padding": "10px 25px",
-    'padding': '10px 25px',
-    "padding-top": null,
-    "padding-bottom": null,
-    "padding-left": null,
-    "padding-right": null,
-    "width": null,
-    "height": null
+    'align':            'center',
+    'background-color': 'DimGray',
+    'border':           null,
+    'border-top':       null,
+    'border-right':     null,
+    'border-bottom':    null,
+    'border-left':      null,
+    'border-radius':    '3px',
+    'box-shadow':       'none',
+    'color':            '#fff',
+    'font-family':      'Ubuntu,Helvetica,Arial,sans-serif',
+    'font-size':        '16px',
+    'font-style':       null,
+    'font-weight':      '400',
+    'line-height':      '21px',
+    'margin':           '0 0 15px 0',
+    'margin-top':       null,
+    'margin-right':     null,
+    'margin-bottom':    null,
+    'margin-left':      null,
+    'padding':          '10px 25px',
+    'padding-top':      null,
+    'padding-right':    null,
+    'padding-bottom':   null,
+    'padding-left':     null,
+    'text-align':       'center',
+    'text-decoration':  'none',
+    'text-transform':   'none',
+    'width':            '100%'
   }
-}
-const baseStyles = {
+};
+const baseStyles            = {
   table: {
-    borderCollapse: 'separate'
+    borderCollapse: 'collapse',
+    borderSpacing:  '0',
+    padding:        '0',
+    textAlign:      'left'
+  },
+  tr:    {
+    padding:   0,
+    textAlign: 'left'
+  },
+  td:    {
+    border:         'none',
+    borderCollapse: 'collapse',
+    hyphens:        'auto',
+    margin:         '0',
+    MozHyphens:     'auto',
+    padding:        '0',
+    WebkitHyphens:  'auto',
+    wordWrap:       'break-word'
   },
-  a: {
-    textDecoration: 'none',
-    lineHeight: '100%'
+  typo:  {
+    margin:  '0',
+    padding: '0'
+  },
+  p:     {
+    margin: '0'
+  },
+  a:     {
+    display:       'block',
+    textTransform: 'none'
   }
-}
+};
 
 @MJMLElement
 class Button extends Component {
 
-  styles = this.getStyles()
+  styles = this.getStyles();
 
   getStyles () {
-    const { mjAttribute, defaultUnit } = this.props
+    const {mjAttribute} = this.props;
 
     return helpers.merge({}, baseStyles, {
       table: {
-        width: mjAttribute('width')
+        margin:       mjAttribute('margin'),
```

Comment:
Wait what!? At first I didnt believe it, but it seems to be true indeed https://www.emailonacid.com/blog/article/email-development/outlook.com-does-support-margins